package.json

```diff
@@ -28,24 +28,30 @@
     "url": "http://www.bentley.com"
   },
   "devDependencies": {
-    "@bentley/build-tools": "^2.0.0",
-    "@bentley/extension-webpack-tools": "^2.0.0",
+    "@bentley/build-tools": "^2.8.0",
+    "@bentley/extension-webpack-tools": "^2.8.0",
+    "@bentley/eslint-plugin": "^2.8.0",
     "cpx": "1.5.0",
+    "eslint": "^6.8.0",
```

Comment:
do we want to wait to bump this to ^7.1.0 after cra update goes through or?

Reply:
Yes, good idea. I'll hold off until that's published.